e2e/pages/adf/dialog/contentList.js

```diff
@@ -308,6 +308,11 @@ var ContentList = function () {
         return this;
     };
 
+    this.getNodeIdFirstElement = function () {
+        let firstNode = element.all(by.css('adf-datatable div[title="Node id"] span')).first();
```

Comment:
This element seems to be part of datatable component. I suggest you to move this method to dataTablePage.js file.